### PR TITLE
Job to cleanup image streams

### PR DESCRIPTION
**Pull request by csrwng**
Pipeline configured to run once a day to clean up image streams.

### CODE REVIEW

**Comment by csrwng** [comment]
@stevekuznetsov

**Comment by kargakis** [comment]
Any reason to not setting owner references from pods to image streams?

**Comment by csrwng** [comment]
the image streams may be used by many things, including e2e tests which are not present yet

File: cluster/ci/origin/pipelines/cleanup-image-streams/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+pipeline {
+  agent any
+  stages {
+    stage("Cleanup ImageStreams") {
+      steps {
+        script {
+          openshift.withCluster() {
+            def cleanupList = []
+            def currentSecsStr = sh(script: "date +%s", returnStdout: true)
+            def currentSecs = currentSecsStr as Integer
```
Comment (stevekuznetsov) [inline cluster/ci/origin/pipelines/cleanup-image-streams/Jenkinsfile]: wtf

File: cluster/ci/origin/pipelines/cleanup-image-streams/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            def imageStreams = openshift.selector("is")
+            imageStreams.withEach() {
+              def imageStream = it.object()
+              // Only process image streams that are part of a build
+              if (imageStream.metadata.labels != null && imageStream.metadata.labels.template == "output-imagestream") {
+                def creationTimestamp = imageStream.metadata.creationTimestamp
+                def imageStreamCreationSecsStr = sh(script: "date --date=${creationTimestamp} +%s", returnStdout: true)
+                def imageStreamCreationSecs = imageStreamCreationSecsStr as Integer
+                def ageDays = (currentSecs - imageStreamCreationSecs) / 3600 / 24
```
Comment (stevekuznetsov) [inline cluster/ci/origin/pipelines/cleanup-image-streams/Jenkinsfile]: Maybe

```sh
$ date --date '2 days ago' +%s
1509818640
```

And a simple comparison between creation time?

Comment (csrwng) [reply cluster/ci/origin/pipelines/cleanup-image-streams/Jenkinsfile]: sgtm, will update

**Pull request merged**